### ISSUE TITLE
Accessibility bug: the nav elements across the theme should have an aria-label

### ISSUE THREAD

**Issue by manish10** [mmistakes/minimal-mistakes#4575]
### What happened?

- Minimal Mistakes version: minimal-mistakes-jekyll-4.24.0
- Ruby gem or remote theme version: 3.2.0
- Jekyll version: whatever github uses
- Git repository URL: https://github.com/IAccessible/IAccessible.github.io.git
- Hosted on GitHub Pages (if yes provide URL to site): yes, https://iaccessible.net
- Operating system: win 11

### Expected behavior

If there are multiple nav landmarks on a Web page, each nav landmark should have a useful aria label. This allows screen reader users to identify what each navigation landmark contains.

### Steps to reproduce the behavior

actual result: None of the nav elements, including the skip links, the top nav in the masthead, and the previous/next nav in the blog post layout have an aria label or aria-labelledby in them to help identify the landmark. This is a WCAG violation.
All the sample pages for minimal mistakes also repro this problem.
